tests/unit/components/Tutorial/Assessments.spec.js

```diff
@@ -301,4 +318,22 @@ describe('success slot for completed assessment', () => {
     expect(message.exists()).toBe(true);
     expect(message.text()).toBe('🍺');
   });
+
+  it('renders a "success" slot message when provided on aria live element for AX', () => {
```

Comment:
couldnt this assertion be added to the above test? Looks like its just asserting a text, everything else is the same, no?

Reply:
Fixed!